ci-operator/step-registry/gcp/provision/bastionhost/gcp-provision-bastionhost-commands.sh

```diff
@@ -19,26 +19,26 @@ if [[ -s "${SHARED_DIR}/xpn.json" ]]; then
   CONTROL_PLANE_SUBNET="$(jq -r '.controlSubnet' "${SHARED_DIR}/xpn.json")"
 fi
 
-if [[ -z "${NETWORK}" || -z "${CONTROL_PLANE_SUBNET}" ]] && [[ ! -s "${SHARED_DIR}/customer_vpc_subnets.yaml" ]]; then
-  echo "Lack of VPC info, abort." && exit 1
+if [[ -s "${SHARED_DIR}/customer_vpc_subnets.yaml" ]]; then
```

Comment:
The original logic is, the env vars `NETWORK` & `CONTROL_PLANE_SUBNET` take the priority, then `${SHARED_DIR}/customer_vpc_subnets.yaml`. So?

Reply:
Right, but actually since we introduced [xpn case](https://github.com/openshift/release/pull/43761/files#diff-6a48d98c9b2dcef6989cd50bce579ce17b112ca0089795c838e15cf65204a7edR16-R20), `NETWORK` and `CONTROL_PLANE_SUBNET` would not take the priority in xpn case. So I adjust the non-xpn case here. So that means, after this PR, `xpn.json` and `customer_vpc_subnets.yaml` would take higher priority than `NETWORK` and `CONTROL_PLANE_SUBNET` , the reason of keep `NETWORK` and `CONTROL_PLANE_SUBNET`  is for some case, there is some pre-existing vpc, users can directly call the bastion step in the vpc by overriding the ENV vars.